### DISCUSSION TITLE
Top-3 most annoying things

### DISCUSSION THREAD

**Discussion by neSpecc** [codex-team/editor.js#2660] [Announcements]
Hey there! We're introducing the new discussion format:

Write a short list of **the most annoying things** or bugs in editor at the moment. It will help us prioritising further improvements.

<img width="844" alt="top-3-bugs" src="https://github.com/codex-team/editor.js/assets/3684889/ef785232-c74f-447d-8c56-6fffba00cf77">

Please, be short and stay in a following format:

```
N. Short description (Link to a corresponded Issue)
```

**Do not write about feature requests**. Here is a [separate discussion](https://github.com/codex-team/editor.js/discussions/2661) for them.

### Example
 
1. New block insertion on inline-pasting (#2142)
2. Cross-Block selection breaks (#2167)
3. Lack of types for Tools (#2143)

**Comment by vigneshksaithal** [comment]
1. Markdown support (https://github.com/codex-team/editor.js/issues/709)
2. Lack of types for Tools (https://github.com/codex-team/editor.js/issues/2143)
3. A filter like Notion, because in current filter, it gets out of position if searched.

**Comment by reindert-vetter** [comment]
1. Hide toolbar actions for 1 field. later I need to allow a maximum number of blocks. Then sometimes I have to show it and sometimes I don't. Now I do:
```css
.ce-toolbar__actions--opened {
    display: none;
}
```

**Comment by sethaddison** [comment]
1. Lack of updates/maintenance for blocks built by the EditorJs team. I see these as being part and parcel of EditorJs, and I wish they were updated and maintained.

**Comment by li-tianpei** [comment]
your documentation is really hard to understand, it's English but also with tons grammar issues. I got lost everywhere. It's a very simply lib but somehow you guys made the docs like reading C. Please make it easier.

**Comment by Tomsgu** [comment]
1. Very limited set of tools - wish there were a little bit more options (columns, footnotes,...), a majority of tools from awesome editorjs are just not working in a newest version.
2. Because there is a limited set of tools, I usually have to write those tools myself and I am having a really hard time with a current documentation. It's mostly outdated (for example inline tool doc doesn't even work properly). New changes are usually not documented or well explained.
